edn/src/types.rs

```diff
@@ -248,6 +292,18 @@ impl Ord for Value {
     }
 }
 
+impl PartialOrd for ValueAndSpan {
```

Comment:
This surprises me.  I don't expect `ValueAndSpan` instances with _different_ spans to compare or order equal.

@rnewman, do you?